### DISCUSSION TITLE
Every room presents an error after a while

### DISCUSSION THREAD

**Discussion by raptile** [DigiLive/mushroom-strategy#146] [Q&A] [answered]
At first, when i go to a room like /strategy/cinema it was navigating to the room and everything was presented nicely.
Now, when i do that i get this error for every room:
```
Error loading the view strategy:

TypeError: Cannot read properties of undefined (reading 'domains')
```

I removed the dashboard and started over again with a fresh one.
It was working fine again.
But after a little a couple of hours, the same happened.

What is going on here?

**Comment by DigiLive** [comment]
What's the content of you configuration file?

**Reply by raptile** [reply]
i just removed it again and added a new dashboard.

The content is:
```
strategy:
  type: custom:mushroom-strategy
views: []
```
I have removed everything and started from scratch. (re-add the mushroom-strategy.sj, remove the dashboard source and re-add it, remove the dashboard, create a new empty one and add above configuration again again)
I'll be waiting again for it to fail.

But now it just gets worse and worse when i try to take control so i can edit rooms.
the rooms get numbers for the URL and unnamed as names.
<img width="819" alt="image" src="https://github.com/user-attachments/assets/8d382057-de6b-4536-adf0-4a6f51857eef">

When i remove the dashboard and re-add it again with above code it is fine again:
<img width="566" alt="image" src="https://github.com/user-attachments/assets/f16a414b-3b87-4139-bf41-065042c5d0e9">

<img width="839" alt="image" src="https://github.com/user-attachments/assets/e8e2de29-ebe3-4981-bd05-98c5a2cfc940">


So i have two problems.. 1 it breaks at some point and two, i am unable to get in control because when i take control, the code breaks as well

**Comment by DigiLive** [comment, marked as answer]
> i am unable to get in control because when i take control, the code breaks

The strategy is not meant to take control of.
It's meant to auto-generate a dashboard, based on a yaml configuration.

**Reply by raptile** [reply]
Oke, so it is not possible to add or remove services?
Which configuration file should i edit?

**Comment by DigiLive** [comment]
The yaml file of you dashboard (in the raw configuration editor).
Please consult the Wiki.

[Basic Setup](https://github.com/AalianKhan/mushroom-strategy/wiki#basic-setup)
[Strategy Options](https://github.com/AalianKhan/mushroom-strategy/wiki/Strategy-Options)

**Reply by raptile** [reply]
<del>The first link is to meldkamer. </del>

But thanks for the info :) i created a duplicate so i have bit more control over it. But it is an amazing tool!
